Tests/TuistDependenciesTests/SwiftPackageManager/SwiftPackageManagerInteractorTests.swift

```diff
@@ -1,28 +1,91 @@
+import TSCBasic
 import TuistCore
+import TuistGraph
 import TuistSupport
 import XCTest
 
 @testable import TuistDependencies
+@testable import TuistDependenciesTesting
 @testable import TuistSupportTesting
 
 final class SwiftPackageManagerInteractorTests: TuistUnitTestCase {
     private var subject: SwiftPackageManagerInteractor!
 
+    private var fileHandlerMock: MockFileHandler!
+
+    private var temporaryDirectoryPath: AbsolutePath!
+
     override func setUp() {
         super.setUp()
-        subject = SwiftPackageManagerInteractor()
+
+        do {
+            temporaryDirectoryPath = try TemporaryDirectory(removeTreeOnDeinit: true).path
+        } catch {
+            XCTFail("Failed to setup TemporaryDirectory")
+        }
+
+        fileHandlerMock = MockFileHandler(temporaryDirectory: { self.temporaryDirectoryPath })
+
+        subject = SwiftPackageManagerInteractor(
+            fileHandler: fileHandlerMock
+        )
     }
 
     override func tearDown() {
+        fileHandlerMock = nil
```

Comment:
there's a mock implementation already created in `TuistUnitTestCase`, so I'd delete this